packages/web/src/components/toolbar.tsx

```diff
@@ -20,12 +20,16 @@ import * as React from 'react';
 type ToolbarProps = {
   noShadow?: boolean;
   noMinHeight?: boolean;
+  className?: string;
+  onClick?: (e: React.MouseEvent) => void;
 };
 
 export const Toolbar: React.FC<React.PropsWithChildren<ToolbarProps>> = ({
   noShadow,
   children,
-  noMinHeight
+  noMinHeight,
+  className,
+  onClick,
 }) => {
-  return <div className={'toolbar' + (noShadow ? ' no-shadow' : '') + (noMinHeight ? ' no-min-height' : '')}>{children}</div>;
+  return <div className={'toolbar' + (noShadow ? ' no-shadow' : '') + (noMinHeight ? ' no-min-height' : '') + ' ' + (className || '')} onClick={onClick}>{children}</div>;
```

Comment:
compiling the `className` string like this screams for https://www.npmjs.com/package/clsx to make it more readable. It's just a couple lines of code, we could steal the implementation if we don't want to add a new dependency. What do you think?

Reply:
+1 for that! clsx has 0 dependencies, we also used that in the past a lot, especially helpful with conditions etc.

Reply:
Filed https://github.com/microsoft/playwright/issues/31863 to track this and more.